docs/docs_source/build_test_ship/settings.rst

```diff
@@ -0,0 +1,105 @@
+
+..  _settings:
+
+Microsoft-configured settings
+=============================
+
+..  spelling::
+
+    subdomain
+    wopi
+    qa
+
+Most Office Online behavior is configurable based on properties provided in :ref:`CheckFileInfo`. However, there are
+some Office Online settings that must be changed by Microsoft.
```

Comment:
After thinking about this and testing a few changes locally, I don't think this adds much. The list of settings is displayed directly on the left, so there's no real 'quick linking' benefits.

![image](https://cloud.githubusercontent.com/assets/19589/9775712/300ab350-5706-11e5-96bd-1e3842593eb6.png)